docs/tools/vdb_table/data/turbopuffer.json

```diff
@@ -168,7 +168,7 @@
   },
   "npm_downloads": {
     "value": 0,
-    "source_url": "",
+    "source_url": "https://www.npmjs.com/package/turbopuffer-sdk",
```

Comment:
Is the @ needed like with the other ones?

Reply:
Both styles work - ex [aperturedb](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/data/aperturedb.json) has without @ and [azureai](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/data/azureai.json) has with